nixio/property.py

```diff
@@ -115,6 +115,15 @@ def _create_new(cls, nixparent, h5parent, name, dtype, oid=None):
 
         return newentity
 
+    def append_values(self, data):
+        """Method for appending values to existing values"""
+        arr = np.array(data).flatten('C')
+        ds = self._h5dataset.dataset
```

Comment:
H5DataSet exposes the `resize()` method through the `shape` property setter:
https://github.com/G-Node/nixpy/blob/ef2637cd198ef98b936e0c1211adbf76d8b57943/nixio/hdf5/h5dataset.py#L74